cumulusci/tasks/bulkdata/mapping_parser.py

```diff
@@ -496,6 +495,21 @@ def strip(element: str):
         ):
             return False
 
+        # inject namespaces into the update_key
+        if self.update_key:
+            assert isinstance(self.update_key, Tuple)
```

Comment:
Confirming: `update_key` is a tuple even if it's a single field?

Reply:
Correct. There's a method that guarantees it and this check double-checks that it wasn't turned off by accident.

https://github.com/SFDO-Tooling/CumulusCI/blob/2b27f19e63157941b58ad4929ef2bce8bfd59528/cumulusci/tasks/bulkdata/mapping_parser.py#L112